### ISSUE TITLE
[question] Why this player is not working closely with Nova player

### ISSUE THREAD

**Issue by jerryn70** [moneytoo/Player#158]
I saw your readme describing jointly using of this player with Nova player . So I have downloaded Nova player used with your player but I have noticed 2 things , in Nova the resume menu not showing the correct resume duration and the last played video is not showing in the menu bar. 
So I have tried Nova with another video player called mpv player (mentioned on your readme). But while using that player resume duration and last played video is displayed correctly in Nova .

Here I'm attaching the screenshot with marking so that you can understand what I mean 
![](https://i.imgur.com/26BUpzx.jpg)
![](https://i.imgur.com/zlEmSKd.jpg)

**Comment by moneytoo** [comment]
Nova mentions that "some features will not work (resume, bookmarks, etc.)". I'm not aware of any (Nova) API to handle that.

![device-2021-08-19-080141](https://user-images.githubusercontent.com/1071643/130016233-aecf30ed-f816-44ba-a60d-65b0fab58343.png)

**Comment by jerryn70** [comment]
@moneytoo so that means there is nothing can do from your side , right

**Comment by moneytoo** [comment]
You are right, it works with mpv. Previously I probably didn't look deep enough. I will definitely try supporting that.
